examples/all-clusters-app/esp32/README.md

```diff
@@ -160,73 +160,71 @@ your network configuration. To erase it, simply run.
 
           $ idf.py -p /dev/tty.SLAB_USBtoUART erase_flash
 
-### Setting up Python Controller
+-   Once ESP32 is up and running, we need to set up a device controller to
+    perform
+-   commissioning and cluster control.
```

Comment:
This line is supposed to go with the above bullet point, in the below files as we.

Reply:
done